tests/neg-custom-args/fatal-warnings/i4674.scala

```diff
@@ -0,0 +1,9 @@
+object Testi4674 {
+  def T(x: String) = {
+    x.foreach {
+      case 's' => println("s")
+      case c: Char => println(c) // error
+    }
+  }
+  T("test")
```

Comment:
This is not needed. I would remove it. We try to keep tests as small as possible